<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt5GUIClass</class>
 <widget class="QMainWindow" name="Qt5GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt5GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="3">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>TextEdit</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="radioButton">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>LineEdit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>추가하기</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>삭제하기</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QTextEdit" name="textEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt5gui.qrc"/>
 </resources>
 <connections/>
</ui>
